tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -28,11 +28,12 @@ class EntityInstance(findings_lib.Findings):
   """
 
   def __init__(self, entity, universe):
+    super(EntityInstance, self).__init__()
     self.entity = entity
     self.universe = universe
     self.required_keys = ('id', 'type')
 
-  def _validate_type(self):
+  def _ValidateType(self):
```

Comment:
lower cap for priate methods `_then_lower_cap`

Reply:
The change is breaking because I think pylint does not support multiple naming styles in the same file? https://docs.pylint.org/en/1.6.0/options.html#multiple-naming-styles

Reply:
Resolved to use CamelCase